src/app/qgsanimationexportdialog.h

```diff
@@ -66,6 +66,12 @@ class APP_EXPORT QgsAnimationExportDialog: public QDialog, private Ui::QgsAnimat
     //! Returns the duration of each individual frame
     QgsInterval frameInterval() const;
 
+    //! Returns the time step of each individual frame
+    double frameTimeStep() const;
+
+    //! Returns the time step unit of each individual frame
+    QgsUnitTypes::TemporalUnit frameTimeStepUnit() const;
```

Comment:
See above -- we don't actually need these changes anymore, the QgsInterval frameInterval method will be sufficient